docusaurus.config.js

```diff
@@ -154,15 +154,20 @@ module.exports = {
           editUrl: "https://github.com/okteto/docs/edit/main",
           breadcrumbs: false,
           sidebarPath: require.resolve("./sidebars.js"),
-          lastVersion: "1.16",
+          lastVersion: "1.17",
           versions: {
             current: {
+              label: "1.18",
+              path: "1.18",
+            },
+            1.17: {
               label: "1.17",
-              path: "1.17",
+              path: "/",
+              banner: "none",
             },
             1.16: {
               label: "1.16",
-              path: "/",
+              path: "1.16",
               banner: "none",
```

Comment:
Should we add the `unmaintained` as we had for `1.15`?

Reply:
Yes, good call 👍